assets/i18n/en.json

```diff
@@ -87,6 +87,19 @@
     "I18N_BLOG_DASHBOARD_PAGE_NEW_POST_BUTTON": "New Post",
     "I18N_BLOG_DASHBOARD_TAB_DRAFTS": "Drafts",
     "I18N_BLOG_DASHBOARD_TAB_PUBLISHED": "Published",
+    "I18N_BLOG_HOME_PAGE_BREADCRUMB": "Blog",
+    "I18N_BLOG_HOME_PAGE_NO_RESULTS_FOUND": "Sorry, No results found currently!",
```

Comment:
Sorry, there are no blog posts to show.

(Or similar. Have you talked to Diana or the UX Writing team yet to verify this text? If not, worth doing soonish.)

Reply:
Changed to - Sorry, there are no blog posts to show.
Most of the text comes from the mocks directly. I have verified the twitter sharing text from Isalo. Title and metacontent from Diana. I will talk to Diana for any other remaining text. Thanks!

Reply:
OK sounds good. Make sure you get confirmation from @dchen97 before merging this PR. Thanks!